edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Http/controllers/HttpRequestAuthenticator.cs

```diff
@@ -39,6 +39,7 @@ public async Task<HttpAuthResult> AuthenticateAsync(string deviceId, Option<stri
 
             IClientCredentials clientCredentials;
             X509Certificate2 clientCertificate = await context.Connection.GetClientCertificateAsync();
+            Events.AuthenticationDebug($"deviceId: {deviceId}, moduleId: {moduleId.GetOrElse(string.Empty)}, client certificate: {clientCertificate != null}");
```

Comment:
ignore all changes in this file, those are used for debugging.  Will remove before commit.